test/metabase/pulse/send_test.clj

```diff
@@ -620,181 +615,6 @@
          (is (mt/received-email-body? :rasta #"Manage your subscriptions"))
          (is (mt/received-email-body? "nonuser@metabase.com" #"Unsubscribe")))))))
 
-(defn- get-positive-retry-metrics [^io.github.resilience4j.retry.Retry retry]
```

Comment:
these tests are moved to `metabase.notification.send-test`

Reply:
Would have been nice to move them in a separate commit so it's easier to diff

Reply:
yea, sorry. FWIW, the only change is this line: https://github.com/metabase/metabase/pull/55312/files#r2000562865